src/test/java/pages/LoginPage.java

```diff
@@ -41,8 +41,8 @@ public LoginPage enterPassword(String password) throws InterruptedException {
     }
 
     public HomePage login() throws InterruptedException {
-        enterEmail("demo@class.com");
-        enterPassword("te$t$tudent");
+        enterEmail("nishaabidi@yahoo.com");
+        enterPassword("Mylife#1234");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

```
    @FindBy(css="[type='email']")
    private WebElement emailField;

    @FindBy(css="[type='password']")
    private WebElement passwordField;

    @FindBy(css="[type='submit']")
    private WebElement submitButton;

    @FindBy(css="[data-testid='login-form']")
    private WebElement loginForm;
```

Please, delete  `Thread.sleep(1000); `

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.**

![image](https://user-images.githubusercontent.com/104761482/204558540-a8ce6c2a-00dc-46e9-898c-26fddd9d0da4.png)